src/current/v23.1/goldengate.md

```diff
@@ -0,0 +1,469 @@
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    *** Latest statistics since 2023-06-09 19:06:44 ***
+        Total inserts                              4.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    End of statistics.
+    GGSCI ({host-name} as oggadm1@ORCL) 31>
+
+
+    # In GGSCI for PG
+    cd $OGG_PG_HOME
+    ./ggsci
+    GGSCI ({host-name}) 7> stats REPLICAT RORPSQL
+    Sending STATS request to Replicat group RORPSQL ...
+    Start of statistics at 2023-06-09 19:29:03.
+    Replicating from OGGADM1.TESTTABLE to public.testtable:
+    *** Total statistics since 2023-06-09 19:10:20 ***
+        Total inserts                              4.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    *** Daily statistics since 2023-06-09 19:10:20 ***
+        Total inserts                              4.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    *** Hourly statistics since 2023-06-09 19:10:20 ***
+        Total inserts                              4.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    *** Latest statistics since 2023-06-09 19:10:20 ***
+        Total inserts                              4.00
+        Total updates                              0.00
+        Total deletes                              0.00
+        Total upserts                              0.00
+        Total discards                             0.00
+        Total operations                           4.00
+    End of statistics.
+    GGSCI ({host-name}) 8> 
+    ~~~
+
+## Bulk replication
+
+1. Keep the [Extract process](#set-up-extract) running on Oracle and the [Replicat process](#set-up-replicat) running for PostgreSQL.
+1. In the source database, bulk insert:
+
+    ~~~
+    INSERT INTO OGGADM1.testtable2 (col1, col2)
+    SELECT level + 99, 'Example data'
+    FROM dual
+    CONNECT BY level <= 50000 - 99;
+    ~~~
+
+1. Run the status command on the Extract `ggsci` terminal.
```

Comment:
@ryanluu12345 Do we need to define EPOS2 somewhere?

```suggestion
1. Run the status command on the Extract GGSCI terminal:

    {% include_cached copy-clipboard.html %}
    ~~~ shell
    stats EXTRACT EPOS2
    ~~~
```

Reply:
Oh yes, I think that might be a copy and paste error. I think in these set of docs it shows the set of steps where we created `EPOS`. So this should be `stats EXTRACT EPOS` or if you want to be more general: `stats EXTRACT <name of your extract task>`

Reply:
the first mention of `epos` i can find is the command `edit param epos`. did that step create it?

Reply:
Yes, this `edit param epos` creates the epos file. It actually opens up an editor for you to put in the contents:
```
# Inside the ggsci terminal run these
edit param epos

# This is the resulting output
# Note that RMTHOST is the server's internal host name: "hostname" command
# The MGRPORT is the port that the PostgreSQL GGSCI manager is running on (8200 as defined above)
# The USERIDALIAS is something I created with credential store, but you can use USERID <user>, PASSWORD <password>
EXTRACT epos
USERIDALIAS gg_source
RMTHOST ip-172-31-0-218.ec2.internal, MGRPORT 8200
RMTTRAIL ./dirdat/ab
TABLE OGGADM1.testtable;

# Save and close

# Then run this
view param epos
```